centreon/www/class/config-generate/broker.class.php

```
@@ -616,22 +616,22 @@ private function getInfoDb($string)
                     break;
                 case 'T':
-                    $s_table = $value;
+                    $s_table = (string) $value;
                     break;
                 case 'C':
-                    $s_column = $value;
+                    $s_column = (string) $value;
                     break;
                 case 'F':
-                    $s_filter = $value;
+                    $s_filter = (string) $value;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$s_filter'. (undefined)

(UnusedLocalVariable)